src/test/resources/application-stubbed.properties

```diff
@@ -48,7 +48,7 @@ endpoint.company_name_mongo_alpha.timer=direct:company_name_mongo_alpha_trigger
 endpoint.company_status_mongo_primary.timer=direct:company_status_mongo_primary_trigger
 endpoint.company_status_mongo_alpha.timer=direct:company_status_mongo_alpha_trigger
 endpoint.company_status_mongo_oracle.timer=direct:company_status_mongo_oracle_trigger
-endpoint.insolvency.company_number.timer=direct:insolvency_company_number_trigger
+endpoint.insolvency.company_number.mongo_oracle.timer=direct:insolvency_company_number_mongo_oracle_trigger
```

Comment:
Incorrect period separator: company_number.mongo_oracle should read company_number_mongo_oracle